keyboards/pabile/p18/keymaps/default/keymap.c

```diff
@@ -0,0 +1,42 @@
+#include QMK_KEYBOARD_H
+/* 
+
+
+enable rgb here 
+
+
+*/
+
+
```

Comment:
```suggestion
```
I'd remove this.

Reply:
thank you for looking into this request. i am learning a lot from your comments and suggestions. I apologize that you have to go through all these. thanks once again.

Reply:
Don’t apologize!  I am happy to help.